service/src/main/java/greencity/service/ubs/UBSManagementServiceImpl.java

```diff
@@ -1521,7 +1521,6 @@ public ReasonNotTakeBagDto saveReason(Long orderId, String description, Multipar
         dto.setCurrentUser(order.getUser().getRecipientName() + " " + order.getUser().getRecipientSurname());
         order.setImageReasonNotTakingBags(pictures);
         order.setReasonNotTakingBagDescription(description);
-        order.setOrderStatus(OrderStatus.CANCELED);
```

Comment:
Why you removed this?

Reply:
When we want to update an order and set order status - "NOT TAKING OUT", previously we must to add the reason of "not taking out" order.
On this step (saving the reason of "not taking out" order) our order status changed to "CANCELED".

After that we can not update our order, because it's status is "CANCELED"

On this flow we use two endpoints:
[/ubs/management/save-reason/{id}]
[/ubs/management/update-order-page-admin-info/{id}]

I suggest, that we should not to change order status here on this step [/ubs/management/save-reason/{id}].
Order status will changed here on this step [/ubs/management/update-order-page-admin-info/{id}].

Reply:
I think you should to  set NOT TAKING OUT status.

Reply:
And this is not proper fix.